apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -531,7 +532,10 @@ export function buildMachine(
                 // Heavier paper can fail to eject completely and trigger a jam state even though the paper isn't physically jammed.
                 // To work around this, we avoid ejecting to front. Instead, we present the paper so it's held by the device in a
                 // stable non-jam state. We instruct the poll worker to remove the paper directly from the 'presenting' state.
-                entry: async (context) => await context.driver.presentPaper(),
+                entry: async (context) => {
+                  await logger.log(LogEventId.BlankInterpretation, 'system');
```

Comment:
Was on the fence about this log since technically state transition logging should cover this. But blank interpretation is 1 of 2 flows that requires poll worker intervention and it felt safer to log these explicitly.